sql/age_main.sql

```diff
@@ -122,17 +122,23 @@ CREATE FUNCTION ag_catalog.drop_label(graph_name name, label_name name,
     LANGUAGE c
     AS 'MODULE_PATHNAME';
 
+--
+-- If `load_as_agtype` is true, property values are loaded as agtype; otherwise
+-- loaded as string.
+--
 CREATE FUNCTION ag_catalog.load_labels_from_file(graph_name name,
```

Comment:
This needs to be added to the `age--1.5.0--y.y.y.sql` file

Reply:
Added to the upgrade script. I didn't have to use the `ALTER EXTENSION` commnad to remove dependency first. Because, Postgres does it automatically if DROP command is called from an upgrade script.